src/ui/toast.js

```diff
@@ -53,121 +40,99 @@ export const toastImportantStyles = {
  */
 export let ToastSpecDef;
 
+/** @const {!Object<string, string>} */
+const iframeAttributes = {
+  'frameborder': '0',
+  'scrolling': 'no',
+  'class': 'swg-toast',
+};
 
 /**
  * The class Notification toast.
  */
 export class Toast {
 
   /**
-   * @param {!Window} win
-   * @param {!ToastSpecDef} spec
+   * @param {!../runtime/deps.DepsDef} deps
+   * @param {{'source': string}} args
```

Comment:
`!Object<string, ?>`

Reply:
linter complaining when using Object<string, ?>
when trying to refer: args.source.

Reply:
You have to use `args['source']`. But please refer to my follow up comments on passing all arguments together as a single map here.

Reply:
Ack.